src/Templates/src/maui-blazor/.template.config/template.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "MAUI", "Android", "iOS", "macOS", "Mac Catalyst", "WinUI", "Blazor" ],
+    "identity": "Microsoft.Maui.BlazorSingleProject",
+    "name": ".NET MAUI Blazor App",
```

Comment:
@davidortinau @Redth  Can we also rename the .NET MAUI template to ".NET MAUI App" for simplicity and consistency?

Reply:
Updated.